package.json

```diff
@@ -231,6 +231,7 @@
         "@microsoft/vscode-file-downloader-api": "^1.0.1",
         "@types/tmp": "^0.1.0",
         "@vscode/extension-telemetry": "^0.8.2",
+        "bufferutil": "^4.0.8",
```

Comment:
do we need dependencies ? @Vidya2606  can you please check ?

Reply:
I was getting a dependency error, so I added it! Now I don't see this error after removing. So, removing it now.